service/src/main/java/com/epam/esm/service/validator/CertificateValidator.java

```diff
@@ -18,19 +19,19 @@
     @Override
     public void validate(Certificate certificate) throws ValidationException {
         if (certificate.getName() == null) {
-            throw new ValidationException("Null name!");
+            throw new ValidationException("Null name!", ErrorCodes.CERTIFICATE_VALIDATION_EXCEPTION);
```

Comment:
Change message to "Certificate name must be filled."  and all messages in the bottom should be depended of sense.

Reply:
Fulfilled